services/dynamic_addressbook_service.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+option java_package = "com.hederahashgraph.service.proto.java";
+// <<<pbj.java_package = "com.hedera.hapi.node.token">>> This comment is special code for setting PBJ Compiler java package
+
+import "query.proto";
+import "response.proto";
+import "transaction_response.proto";
+import "transaction.proto";
+
+/**
+ * Transactions for Dynamic Address Book Service
+ */
+service DynamicAddressBookService {
+    /**
+     * Add a new node to address book by submitting the transaction
+     */
+    rpc addNode (Transaction) returns (TransactionResponse);
+
+    /**
+     * Remove a node from address book by submitting the transaction
+     */
+    rpc removeNode (Transaction) returns (TransactionResponse);
+
+    /**
+     * Update a node by submitting the transaction
+     */
+    rpc updateNode (Transaction) returns (TransactionResponse);
```

Comment:
We should consider adding a query to list nodes. In general we want to offload queries to mirror nodes, but I think the list of nodes may be small enough and critical enough to provide such an API.

Reply:
Need to discuss what kind of query.

Reply:
Query will not be in phase 2.